### ISSUE TITLE
Here's the Docs on Installing on Mac Without Steam

### ISSUE THREAD

**Issue by mathwhiz1212** [endless-sky/endless-sky#8451]
### Location

Releases/Wiki

### Describe the issue

It's not clear how to install Endless Sky on a Mac without Steam. The player should first be sent to Steam (easiest) and secondly given a link to a wiki page showing how to install without Steam. 

I wrote a draft of that wiki page and recorded a video tutorial showing how to do the same thing.

# The Problem

The easiest way to install Endless Sky on a Mac is using Steam: https://store.steampowered.com/app/404410/Endless_Sky/

But let’s say you don’t want to install Steam.

The player opens the .dmg file and gets this confusing (potentially scary) message with no clear way to install Endless Sky:

![Screenshot 2023-02-23 at 5 04 34 PM](https://user-images.githubusercontent.com/11021683/221069554-5b7fb0f4-7b0e-4432-85d1-31398693c84e.png)

### Expected content

# Installing without Steam

The video tutorial: https://www.youtube.com/watch?v=za2cA8MKycw

## Download Endless Sky

Go to: https://github.com/endless-sky/endless-sky/releases

Scroll down until you see the version you want, you may want the latest “stable” release:

Left-click on Assets.

Left click on the .dmg file to download it.

Navigate to where you want it and click save.

## Installing Endless Sky

Double-left click on the .dmg file to open it.

Left-click on “EndlessSky” and hold that left-click while you drag it over the folder. 

Once you see the plus icon, you can let go to drop it into the folder.

Left click on the folder (unless the Applications window popped up)

Wait for it to finish copying (it’s done when “Endless Sky” is no longer grayed out).

**Hold down Control** and left click on “EndlessSky”.

**Continue to hold down control** while you click open.

This prompt will tell you the app hasn’t been verified by Apple. That’s why we have to do this.

If it doesn’t work, try again.

Left-click on Open.

# Conclusion

You should now have Endless Sky installed.

**Comment by mOctave** [comment]
I could have sworn this was in the documentation somewhere, although I can't find it. It would definitely be a good idea for instructions regarding this opening unverified Mac apps to appear somewhere, but a lot of it is much simpler than necessary.

**Comment by RisingLeaf** [comment]
I would consider it common knowledge for mac users, but might as well include it in the docs.
@quyykk you area

**Comment by mathwhiz1212** [comment]
> I could have sworn this was in the documentation somewhere, although I can't find it. It would definitely be a good idea for instructions regarding this opening unverified Mac apps to appear somewhere, but a lot of it is much simpler than necessary.

I picked up the habit of explaining things "much simpler than necessary" by doing tutorials at a Minecraft hosting company.

It depends on the target audience.

With techy people, it doesn't matter as much. If you want anyone to be able to play, I've found it best to be as simple as possible.

**Comment by mathwhiz1212** [comment]
> I would consider it common knowledge for mac users, but might as well include it in the docs. @quyykk your area

I'm sure tech-savvy mac users who want to run unsigned software know how, but I didn't until today. I'm tech savvy, but never needed to.

This guide is aimed at Mac users that aren't super tech savvy who want to play this game without installing Steam.

The first solutions I ran across would likely intimidate that audience. I chose to use another computer because I didn't realize how simple it was.

**Comment by RisingLeaf** [comment]
nah, probably just my perspective as I use a lot of third party software that is not signed.
also it can not hurt to have it

**Comment by mathwhiz1212** [comment]
> nah, probably just my perspective as I use a lot of third party software that is not signed. also it can not hurt to have it

Makes sense.

**Comment by quyykk** [comment]
Maybe we should look into code signing the executable? I believe Windows users get a similar warning too.

**Comment by RisingLeaf** [comment]
For windows? sure
mac needs a dev account and you need to pay fees so...

**Comment by RisingLeaf** [comment]
oh, wait, actually it does not cost money, we still had to make a dev account for ES though

**Comment by mathwhiz1212** [comment]
I think you need an OV cert to sign code for Windows, though that's going off what I remember of a project I did a couple years ago.

If money is an issue, you could offer a "self-serve version" for free and charge $0.99 in the Windows & Mac stores just to recoup the cost of putting the apps there. Not sure if that would fit what you have going with ES though.

**Comment by mathwhiz1212** [comment]
> Maybe we should look into code signing the executable? I believe Windows users get a similar warning too.

Long-term that would be preferable and can be automated. It does cost a little money though last I knew.

The warning wasn't as much the issue for me, it's that there wasn't a "do it anyway" button.

**Comment by RisingLeaf** [comment]
> I think you need an OV cert to sign code for Windows, actually signing code was pretty simple. Though that's going off what I remember of a project I did a couple years ago.
> 
> If you end up doing that, hit me up. I might have a tool to help.
> 
> If money is an issue, you could offer a "self-serve version" for free and charge $0.99 in the Windows & Mac stores just to recoup the cost of putting the apps there. Not sure if that would fit what you have going with ES though.

Money management if not done by a single person  would require making this project a type of company or have it handled by sip, I don't think we want that.

**Comment by mathwhiz1212** [comment]
> > I think you need an OV cert to sign code for Windows, actually signing code was pretty simple. Though that's going off what I remember of a project I did a couple years ago.
> > If you end up doing that, hit me up. I might have a tool to help.
> > If money is an issue, you could offer a "self-serve version" for free and charge $0.99 in the Windows & Mac stores just to recoup the cost of putting the apps there. Not sure if that would fit what you have going with ES though.
> 
> Money management if not done by a single person would require making this project a type of company or have it handled by sip, I don't think we want that.

Exactly. It sounds like you'd need an individual willing to spend money out of their own pocket or just tell people how to work around it.

**Comment by mOctave** [comment]
> Run unsigned software? Never needed to. This guide is aimed at Mac users that aren't super tech savvy who want to play this game without installing Steam.

Just to be clear, I think the part about how you can open unsigned software is the important part, and should be kept. It's the rest of it that might not be necessary – if someone is using GitHub, they presumably know how to download files and install an app, or are getting help from someone who does know how. It probably wouldn't hurt to include it in the docs, but the most important steps are just:

1. Get the .dmg from whatever release you want.
2. You will need to control-click->open or right-click->open, possibly multiple times, in order to get the app to launch.
3. Don't worry about Apple's warning.

The rest of the instructions probably don't matter too much, although it might just be me.

**Comment by mathwhiz1212** [comment]
> > Run unsigned software? Never needed to. This guide is aimed at Mac users that aren't super tech savvy who want to play this game without installing Steam.
> 
> Just to be clear, I think the part about how you can open unsigned software is the important part, and should be kept. It's the rest of it that might not be necessary – if someone is using GitHub, they presumably know how to download files and install an app, or are getting help from someone who does know how. It probably wouldn't hurt to include it in the docs, but the most important steps are just:
> 
> 1. Get the .dmg from whatever release you want.
> 2. You will need to control-click->open or right-click->open, possibly multiple times, in order to get the app to launch.
> 3. Don't worry about Apple's warning.
> 
> The rest of the instructions probably don't matter too much, although it might just be me.

Those are certainly the most important points and for you or I that would be all that is needed.

The rest of it is just in case there's someone who isn't as knowledgeable who doesn't want to install Steam.

It sounds like I've had more experience with people who don't easily grasp tech, and you've had more experience with those who pick it up quickly. 

That seems like the difference in perspective on this.

**Comment by mOctave** [comment]
> It sounds like I've had more experience with people who don't easily grasp tech, and you've had more experience with those who pick it up quickly.
>
> That seems like the difference in perspective on this.

That's fair. My presumption is that if someone is on GitHub, they probably understand how to do the rest of it, but, seeing as how there are differences in perspective and it's probably better too be safe than sorry, I'd honestly be okay with the full instructions as long as they were either in a separate file for mac installation or under a big enough heading so that they wouldn't confuse people looking for Windows/Linux instructions or make it very hard for WIndows/Linux users to be able to quickly find instructions for their operating systems.

**Comment by mathwhiz1212** [comment]
> > It sounds like I've had more experience with people who don't easily grasp tech, and you've had more experience with those who pick it up quickly.
> > That seems like the difference in perspective on this.
> 
> That's fair. My presumption is that if someone is on GitHub, they probably understand how to do the rest of it, but, seeing as how there are differences in perspective and it's probably better too be safe than sorry, I'd honestly be okay with the full instructions as long as they were either in a separate file for mac installation or under a big enough heading so that they wouldn't confuse people looking for Windows/Linux instructions or make it very hard for WIndows/Linux users to be able to quickly find instructions for their operating systems.

That makes sense.

I 100% agree that making it easier for one OS shouldn't make it harder for others.
